scripts/rollup/bundles.js

```diff
@@ -318,7 +309,7 @@ const bundles = [
     ],
   },
   {
-    bundleTypes: [FB_WWW_DEV, FB_WWW_PROD],
+    bundleTypes: [], // TODO: FB_WWW_DEV, FB_WWW_PROD
```

Comment:
Don't delete this please. `ReactFlightDOMRelayServerRuntime-dev.classic.js` is unused but `ReactFlightDOMRelayServerRuntime-dev.modern.js` is used.

Reply:
Ahhh. I get it now. Let me roll that back.

Reply:
fixed